Doc/library/heapq.rst

```diff
@@ -110,17 +110,17 @@ The module also offers three general purpose functions based on heaps.
 
    Return a list with the *n* largest elements from the dataset defined by
    *iterable*.  *key*, if provided, specifies a function of one argument that is
-   used to extract a comparison key from each element in the iterable:
-   ``key=str.lower`` Equivalent to:  ``sorted(iterable, key=key,
+   used to extract a comparison key from each element in the iterable (for example,
+   ``key=str.lower``).  Equivalent to:  ``sorted(iterable, key=key,
    reverse=True)[:n]``
```

Comment:
Isn't a period missed?

Reply:
You mean at the very end. Not sure about this because that last part is not a complete sentence, but then I'm not a native speaker :man_shrugging:

Reply:
FWIW, I would probably use a full stop at the end (and again for _nsmallest_). The last part still functions as a sentence, it comes after other sentences (now) using full stops, and another full stop would identify the end of the paragraph.

If the code was written on a separate line, I probably wouldn’t use a full stop.

Reply:
Sounds convincing, plus the docs have many other examples of incomplete sentences like this that all have a period.